bioseq_tools/bio_files_processor.py

```diff
@@ -0,0 +1,60 @@
+def convert_multiline_fasta_to_oneline(input_fasta: str, output_fasta: str = None):
+    """
+    Converts a multiline FASTA file to oneline FASTA file
+
+    Args:
+        input_fasta: path to the input FASTA file
+        output_fasta: path to the output FASTA file (optional. If not provide, output file will have the same name as the input file with addition “_oneline.fasta”)
+
+    Return:
+        None
+    """
+
+    if output_fasta is None:
+        output_fasta = f"{input_fasta.rsplit('.', 1)[0]}_oneline.fasta"
+
+    with open(input_fasta, "r") as infile, open(output_fasta, "w") as outfile:
+        sequence = ""
+        header = ""
+        for line in infile:
+            line = line.strip()
+            if line.startswith(">"):
+                if sequence:
+                    outfile.write(f"{header}\n{sequence}\n")
```

Comment:
Interesting way to use f-strings in the function, I would have never thought, but they make perfect sense though FASTA being text.